registry.json

```diff
@@ -1,4 +1,20 @@
 [
+    {
+        "ticker": "KNX",                                                      
+        "poolId": "6dbddde53694028eda7ddf85633650a917f6d5e96da97be88b4d4436",
+        "social": {                                                            
+            "website": "https://cryptoknox.github.io/cryptoknox/",
+            "discord": "",
+            "reddit": "",
+            "instagram": " ",
+            "facebook": " ",  
+            "youtube": " ",
+            "twitter": "yyoounas "
```

Comment:
Hi @cryptoknox - please can you add some activity to the twitter? The reason we ask for a social media presence is to see some evidence you can communicate with your delegates.